src/actions.rs

```diff
@@ -609,6 +618,28 @@ impl Action {
             .churn_needed
             .map_or(false, |v| v == ChurnNeeded::Split)
     }
+
+    pub fn complete_merge(&self) {
+        self.0.borrow_mut().complete_merge()
+    }
+
+    pub fn has_sibling_merge_info(&self) -> bool {
+        match self.0.borrow().merge_infos {
+            Some(merge_info) => {
+                let our_section = self.our_section().0;
+                let their_section = merge_info.0;
+                (our_section.0 - their_section.0).abs() == 1
```

Comment:
Because we create a convention here, probably better to add a comment for this function explaining it.

Reply:
Also, I think SectionInfo.1 currently is pretty much the version of the Section info. and SectionInfo.1 the prefix of the section.
Maybe worth clarifying thing by changing that:

maybe :
```
struct SectionInfo {
   // I'm not 100% sure, but it seem to be the section prefix, so that is good that you updated it here.
   prefix: SectionPrefix,
   // The section version: Stand in for members info
   version: i32
}
```

I think it may help finding an easier convention to understand.

Reply:
Currently `(SectionInfo.0).0` is just an `i32` so it's just a number giving the section a name right, not a prefix?

Reply:
Yes, but a number could also be a prefix.
Before merge/split, it did not matter very much.

I guess I'm not sure what are the convention that we'll be using for both split & merge.
Maybe function comment will be enough.

Reply:
I'd expect a prefix to be unsigned though? I'll add comments explaining the current convention and if we want to switch to prefixes let's do that as a follow up PR as I'd expect it to touch most of the code?

Reply:
Sounds good.